core/templates/dev/head/pages/library/Library.js

```diff
@@ -27,19 +27,23 @@ oppia.constant('LIBRARY_PAGE_MODES', {
 });
 
 oppia.controller('Library', [
-  '$scope', '$http', '$log', '$uibModal', '$rootScope', '$window', '$timeout',
-  'ConstructTranslationIdsService', 'UrlService', 'ALL_CATEGORIES',
-  'SearchService', 'WindowDimensionsService', 'UrlInterpolationService',
-  'LIBRARY_PAGE_MODES', 'LIBRARY_TILE_WIDTH_PX', 'AlertsService',
-  'LearnerDashboardIdsBackendApiService', 'UserService',
-  'LearnerDashboardActivityIdsObjectFactory', 'LearnerPlaylistService',
+  '$http', '$log', '$rootScope', '$scope', '$timeout', '$uibModal', '$window',
```

Comment:
If I'm not mistaken, I believe `$scope` should come first, then the rest comes in alphabetical order.

Reply:
gHi @kevinlee12, as @brianrodri suggested [here](https://github.com/oppia/oppia/issues/6004#issue-392503295). All the sorted dollar imports should come first, then regular ones, and then the constant ones.

Reply:
noted, though I'm pointing this out since it's a (implicit) convention that is typically followed, so this would change that convention.

Reply:
/cc @oppia/dev-workflow-team 

What should the convention be? We should fix it and have a lint check to enforce it.

Reply:
Hi @seanlip, I have refactor all the files to introduce this convention.

Reply:
Just to be clear, I am fine with any convention that the dev workflow team agrees on. I just want to suggest that the convention be clearly stated [here](https://github.com/oppia/oppia/wiki/Coding-style-guide), and that there's a lint check for it.

@apb7, as lead of the dev workflow team, could you please weigh in?

Reply:
@seanlip, I am fine with this convention as stated here: https://github.com/oppia/oppia/issues/6004#issue-392503295. This PR refactors the files as well as adds a lint check for the above-mentioned convention.
@anubhavsinha98, can you please add this to the coding style guide here: https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript
Thanks!

Reply:
Okay sure @apb7 !

Reply:
Done